images/macos/macos-12-Readme.md

```diff
@@ -308,29 +308,29 @@ mv "/Applications/Visual Studio 2019.app" "/Applications/Visual Studio.app"
 | Android SDK Platforms      | android-33 (rev 2)<br>android-32 (rev 1)<br>android-31 (rev 1)<br>android-30 (rev 3)<br>android-29 (rev 5)<br>android-28 (rev 6)<br>android-27 (rev 3) |
-| Android SDK Platform-Tools | 33.0.2                                                                                                                                                 |
+| Android SDK Platform-Tools | 33.0.3                                                                                                                                                 |
 | Android Support Repository | 47.0.0                                                                                                                                                 |
 | CMake                      | 3.18.1<br>3.22.1                                                                                                                                       |
 | Google Play services       | 49                                                                                                                                                     |
 | Google Repository          | 58                                                                                                                                                     |
-| NDK                        | 23.2.8568313<br>24.0.8215888<br>25.0.8775105 (default)                                                                                                 |
+| NDK                        | 23.2.8568313<br>24.0.8215888<br>25.1.8937393 (default)                                                                                                 |
 | SDK Patch Applier v4       | 1                                                                                                                                                      |
 
 #### Environment variables
 | Name                    | Value                                              |
 | ----------------------- | -------------------------------------------------- |
 | ANDROID_HOME            | /Users/runner/Library/Android/sdk                  |
-| ANDROID_NDK             | /Users/runner/Library/Android/sdk/ndk/25.0.8775105 |
-| ANDROID_NDK_HOME        | /Users/runner/Library/Android/sdk/ndk/25.0.8775105 |
-| ANDROID_NDK_LATEST_HOME | /Users/runner/Library/Android/sdk/ndk/25.0.8775105 |
-| ANDROID_NDK_ROOT        | /Users/runner/Library/Android/sdk/ndk/25.0.8775105 |
+| ANDROID_NDK             | /Users/runner/Library/Android/sdk/ndk/25.1.8937393 |
+| ANDROID_NDK_HOME        | /Users/runner/Library/Android/sdk/ndk/25.1.8937393 |
+| ANDROID_NDK_LATEST_HOME | /Users/runner/Library/Android/sdk/ndk/25.1.8937393 |
+| ANDROID_NDK_ROOT        | /Users/runner/Library/Android/sdk/ndk/25.1.8937393 |
 | ANDROID_SDK_ROOT        | /Users/runner/Library/Android/sdk                  |
 
 ### Miscellaneous
-- libXext 1.3.4
-- libXft 2.3.4
-- Tcl/Tk 8.6.12
-- Zlib 1.2.12
+- libXext stable
```

Comment:
We should check it.

Reply:
<img width="604" alt="image" src="https://user-images.githubusercontent.com/88318005/186608863-8001e967-1098-4be6-a461-bf2574f88d09.png">

Reply:
so it is shifted for a bit, lets change the generator

Reply:
issue created https://github.com/actions/runner-images-internal/issues/4231